_data-prepper/pipelines/configuration/processors/obfuscate.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: obfuscate
+parent: Processors
+grand_parent: Pipelines
+nav_order: 71
+---
+
+# obfuscate
+
+The `obfuscate` process enables obfuscation to fields inside your documents in order to protect sensitive data. 
+
+## Usage
+
+In this example, we have a document that contains a `log` field and a `phone` field. We want to obfuscate the data in both fields, as shown in the following object:
```

Comment:
For procedures or instructions, ensure that action is taken by the user (“Then you can stop the container...”) rather than the writer (“We also have to stop the container...”).